src/serializer.ts

```diff
@@ -17,14 +19,157 @@ interface PaymentJSON {
  * Provides functionality to serialize from protocol buffers to JSON objects.
  */
 class Serializer {
+/**
+   * Convert a Transaction to a JSON representation.
+   *
+   * @param {proto.Transaction} transaction A Transaction to convert.
+   * @returns {Object} The Transaction as JSON.
+   */
+  public static transactionToJSON(
+    transaction: Transaction
+  ): object | undefined {
+    // Serialize the protocol buffer to a JSON representation.
+    var object: any = transaction.toObject();
+
+    // Convert fields names where direct conversion is possible.
+    this.convertPropertyName("sequence", "Sequence", object);
```

Comment:
What is this method doing?

Reply:
This method re-writes the names of fields on an object. The format that rippled expects isn't the widely accepted JSON that the protocol buffer formats itself to. 

Deferring the conversation on whether this is a good idea to the main comment thread.